doc/manual/src/glossary.md

```diff
@@ -85,12 +85,17 @@
 
     [store path]: #gloss-store-path
 
+  - [file system object]{#gloss-store-object}\
+    The Nix data model for representing file system data.
```

Comment:
```suggestion
    The Nix data model for representing file system data, restricted to the essentials.
```

Hint that it doesn't deal with most of the complexity of the file system. This aspect is important enough to make the glossary, but the details are still in the linked page.

```suggestion
    The Nix data model for representing simplified file system data.
```

```suggestion
    The Nix data model for representing file system data, restricted to files, directories, symlinks and the executable bit.
```